dev/tools/make-changelog.sh

```diff
@@ -3,11 +3,13 @@
 echo "PR number"
 read -r PR
 
-echo "Where? (type a prefix, with no trailing slash)"
+echo "Where? (type a prefix)"
 (cd doc/changelog && ls -d */)
 read -r where
 
-where=$(echo doc/changelog/"$where"*)
+where="doc/changelog/$where"
+if ! [ -d "$where" ]; then where=$(echo "$where"*); fi
+where=$(realpath $where)
```

Comment:
Why get the absolute path? Also IIRC we support systems which don't have the `realpath` command.